src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -183,8 +189,63 @@ object DatabricksPlugin extends AutoPlugin {
     }
   }
 
+  private lazy val executeCommandImpl: Def.Initialize[Task[Seq[CommandStatus]]] = Def.taskDyn {
+    Def.task {
+      val client = dbcApiClient.value
+      val language = dbcExecutionLanguage.value
+      val onClusters = dbcClusters.value
+      val (allClusters, _) = dbcFetchClusters.value
+      val commandFile = dbcCommandFile.value
+      val commandStatuses = Seq.empty[CommandStatus]
+      for (clusterName <- onClusters) {
```

Comment:
This wouldn't properly handle `ALL_CLUSTERS` right? Could you please add a test for that as well. You can use something like:
```scala
val onClusters =  getRealClusterList(dbcClusters.value, allClusters)
```